config/kubernetes-sigs/org.yaml

```
@@ -1195,7 +962,6 @@ teams:
   kubernetes/sig-scheduling:
     description: Team responsible for sig-scheduling related projects
     members:
```

```suggestion
  kubernetes/sig-scheduling:
    description: Team responsible for sig-scheduling related projects
    members:
```

With the removal, `kubernetes/sig-scheduling` team is now empty.

In a new separate commit - let's add SIG Scheduling leads - https://github.com/kubernetes/community/tree/master/sig-scheduling#leadership

---

This is done
Refer commit 23daf5c8fb15374c933f97e49a2b8a2bec45606a
Please note: Skipped adding dom4ha because they aren’t a member of the kubernetes-sigs org